### ISSUE TITLE
Missing link for vcenter_ovf_libraryitem docs

### ISSUE THREAD

**Issue by ericcames** [ansible-collections/vmware.vmware_rest#526]
### Summary

When I try this link: https://github.com/ansible-collections/vmware.vmware_rest/blob/main/docs/vmware.vmware_rest.vcenter_ovf_libraryitem_module.rst I am getting a 404 error

### Issue Type

Documentation Report

### Component Name

vmware.vmware_rest.vcenter_ovf_libraryitem_module.rst

### Ansible Version

```console (paste below)
$ ansible --version

```

### Collection Versions

```console (paste below)
$ ansible-galaxy collection list
```

### Configuration

```console (paste below)
$ ansible-config dump --only-changed

```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by bardielle** [comment]
It was removed from the README file as part of that [PR](https://github.com/ansible-collections/vmware.vmware_rest/pull/522/files)
I can't find this link in our repo

**Comment by mariolenz** [comment]
@ericcames Since the collection is part of the Ansible Community Package, you can find the documentation [here](https://docs.ansible.com/ansible/devel/collections/vmware/vmware_rest/index.html). Please note that this is the documentation for the collection version that's part of the latest Ansible release, which might not be the one you're using.

Additionally, you can find it on [galaxy](https://galaxy.ansible.com/ui/repo/published/vmware/vmware_rest/docs/?version=4.2.0). I think this is quite new. This might make it easier for you to find the documentation for the collection version you're using.

Of course, there's always `ansible-doc vmware.vmware_rest.vcenter_ovf_libraryitem` as an offline alternative.
